src/ts/components/Main.ts

```diff
@@ -0,0 +1,32 @@
+import { createNode } from '../domHelper';
+import store from '../store';
+import { Component } from '../types';
+import SaveVideoSection from './SaveVideoSection';
+import Video from './Video';
+
+interface Props {}
+
+const Main: Component<Props> = ({}) => {
+  const state = store.getState();
+
+  const $main = createNode('<main class="mt-10"></main>', [
+    SaveVideoSection({
+      children: state.saveVideoList.map((video, index) =>
+        Video({
+          type: 'save',
+          channelId: video.snippet.channelId,
+          channelName: video.snippet.channelTitle,
+          publishTime: video.snippet.publishTime,
+          videoId: video.id.videoId,
+          title: video.snippet.title,
+          index,
+          isSave: true,
```

Comment:
```suggestion
        Video({
          type: 'save',
          video,
          index,
          isSave: true,
```
이렇게 아예 video 객체를 넘겨주는 방식은 어떨까요?